static/global.css

```diff
@@ -2,7 +2,32 @@
 
 @font-face {
   font-family: 'Visby CF';
-  src: url('fonts/VisbyCF-Bold.woff2');
+  font-display: block;
+  font-weight: 700;
+  src: url('fonts/VisbyCF-Bold.woff2') format('woff2'),
+    url('fonts/VisbyCF-Bold.woff') format('woff');
+}
+
+@font-face {
+  font-family: 'Open Sans';
+  font-weight: 400;
+  font-display: block;
+  src: url('fonts/OpenSans-Regular.ttf');
```

Comment:
Given that you're already using Typekit/Adobe Fonts for "interstate", I'd suggest you do the same for Open Sans. They deliver the most optimal file format based on the user agent, and it keeps blobs out of your repo.

Reply:
My Adobe account expires soon, so we'd use Google Fonts instead. Decided to do this to edit the `font-display` property, but I'm fine with transferring over if you think it's best